.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "venv/bin/python"
+}
```

Comment:
For future reference, we'll be adding all files specific to your coding environment to a .gitignore file. This will prevent any conflicts your code may have with another teammates' coding environment.